sticky_notes.py

```diff
@@ -21,17 +21,19 @@ def stickynote(self, event):
         if user not in self.notes:
             self.notes[user] = []
 
-        self.notes[user].append("Note from {0}: {1}".format(user, content))
+        notes = self.notes[user]
+        notes.append("Note from {0}: {1}".format(event.meta['nick'], content))
+        self.notes[user] = notes
```

Comment:
The real reason is that this is a `shelf`, not a normal `dict`, that is, it is being saved to disk every time.